jpegexport.py

```diff
@@ -27,20 +27,20 @@
 import inkex
 import simpletransform
 
-inkex.localize()
+inkex.localization.localize
 
 class JPEGExport(inkex.Effect):
 
     def __init__(self):
         """Init the effect library and get options from gui."""
         inkex.Effect.__init__(self)
 
-        self.OptionParser.add_option("--path",    action="store", type="string",  dest="path",    default="",        help="")
-        self.OptionParser.add_option("--bgcol",   action="store", type="string",  dest="bgcol",   default="#ffffff", help="")
-        self.OptionParser.add_option("--quality", action="store", type="int",     dest="quality", default="90",      help="")
-        self.OptionParser.add_option("--density", action="store", type="int",     dest="density", default="90",      help="")
-        self.OptionParser.add_option("--page",    action="store", type="inkbool", dest="page",    default=False,     help="")
-        self.OptionParser.add_option("--fast",    action="store", type="inkbool", dest="fast",    default=True,      help="")
+        self.arg_parser.add_argument("--path",    action="store", type=str,           dest="path",    default="",        help="")
```

Comment:
action="store" and type=str can be removed. A help text would be quite useful, though. It will appear when the extension is used from the command line.